src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService+Protocol.swift

```diff
@@ -25,16 +25,25 @@ protocol ExposureSubmissionService: class {
 	typealias TestResultHandler = (Result<TestResult, ExposureSubmissionError>) -> Void
 	typealias TANHandler = (Result<String, ExposureSubmissionError>) -> Void
 
+	var devicePairingConsentAcceptTimestamp: Int64? { get }
+	var devicePairingSuccessfulTimestamp: Int64? { get }
+
 	func submitExposure(completionHandler: @escaping ExposureSubmissionHandler)
 	func getRegistrationToken(
 		forKey deviceRegistrationKey: DeviceRegistrationKey,
 		completion completeWith: @escaping RegistrationHandler
 	)
 	func getTestResult(_ completeWith: @escaping TestResultHandler)
+
+	/// Fetches test results for a given devide key.
+	///
+	/// - Parameters:
+	///   - deviceRegistrationKey: the device key to fetch the test results for
+	///   - useStoredRegistration: flag to show if a separate registration is needed (`false`) or an existing registration token is used (`true`)
+	///   - completion: a `TestResultHandler`
```

Comment:
The preferred format should be 'docs in protocol' rather than implementation. Changing this for existing code is subject to a different PR.